src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    public String address;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.